web/client/components/maps/enhancers/enhancers.js

```diff
@@ -91,7 +91,7 @@ export const withEmptyMapVirtualScrollProperties = ({ loadPage: lp, scrollSpyOpt
     loadPage: emptyMap(lp),
     hasMore: ({total, items}) => {
         if (items && items.length >= 1 && items[0].id === "EMPTY_MAP") {
-            return total > (items.length + 1);
+            return total > (items.length - 1);
```

Comment:
Didn't get this change

Reply:
if the total of resources from the server is 11 and the items per page are 10 the first request will return 10 items + 1 added locally as empty map. So I think we should remove 1 from the item length that represent the local empty map.

We could add 1 but I should to the total instead, either ways should work.

Reply:
Ok, so it's a bug fix. Now I get it and it seems correct
- total: 11
- items: 11 (10 + 1)
- 11 > 11 - 1